File: ExecutionContext/MultipleOrderedEC-C++/FEComp.cpp
/*!
* @file  FEComp.cpp
* @brief 直列ブロックのウィジェット
*
*/

#include <QtWidgets>
#include "FEComp.h"




using namespace RTC;
using namespace std;


/**
*@brief 直列ブロックのウィジェットのコンストラクタ
* @param parent 親ウィジェット
*/
FEComp::FEComp(QWidget *parent)
    : QWidget(parent)
{
	this->CL = new QVBoxLayout();
	this->setLayout(this->CL);
}

File: ExecutionContext/MultipleOrderedEC-Python/MultipleOrderedEC_Lib/FEComp.py
# -*- coding: utf-8 -*-
##
#   @file FEComp.py
#   @brief 直列ブロックのウィジェット

from PyQt4 import QtCore, QtGui



##
# @class FEComp
# @brief 直列ブロックのウィジェット
#
class FEComp(QtGui.QWidget):
    ##
    # @brief コンストラクタ
    # @param self 
    # @param parent 親ウィジェット
    def __init__(self, parent=None):
        super(FEComp, self).__init__(parent)
        self.ECS = []
        self.CL = QtGui.QVBoxLayout()
        
        self.setLayout(self.CL)

        
